components/table/table-wrapper.js

```diff
@@ -190,6 +214,15 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				reflect: true,
 				type: Boolean
 			},
+			/**
+			 * Whether to wrap the table in a scroll-wrapper (enabled by default when sticky-headers is false).
+			 * @type {boolean}
+			 */
+			scrollWrapper: {
```

Comment:
Wondering if this should be called something like `sticky-headers-scroll-wrapper`? Just due to the confusion that by default tables get a scroll wrapper _unless_ they have sticky headers. And even that was only because we didn't initially support both, but now that we do... yeah it's confusing. :)

Reply:
Yeah, I debated this. Options:

1. ❌ No extra flag at all, change `sticky-headers` to always use this new mode
    - doesn't work because the new mode doesn't support all existing sticky behaviour (header rows that aren't in thead), and because some use cases might specifically not want scroll-wrappers
2. ❌ No extra flag at all, change the default (non-`sticky-headers`) to always use the new mode
    - pretty unintuitive, because sticky behaviour would be enabled without the `sticky-headers` flag
    - also there might use cases that specifically don't want any sticky behaviour
3. ❓ A single `sticky-headers-scroll-wrapper` flag, used *instead of* `sticky-headers`
    - definitely an option; I wondered if this would be more confusing though
    - also the CSS is a bit messier this way, need to double most checks for all the shared logic like `d2l-table-wrapper[sticky-headers] ..., d2l-table-wrapper[sticky-headers-scroll-wrapper] ...`
4. ⭐ Add an extra `scroll-wrapper` flag used in combo with `sticky-headers`
    - this is what I opted for...
    - could name this more verbosely, I thought it would be overkill to use e.g. `<d2l-table-wrapper sticky-headers sticky-headers-scroll-wrapper>`
    - but yes, it's confusing to people not using sticky-headers, because in that case specifying `scroll-wrapper` makes no difference (it's always enabled)